lmdeploy/pytorch/configurations/deepseek_vl2.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from .builder import AutoModelConfigBuilder
+from .default import DefaultModelConfigBuilder
+
+
+class DeepseekVLV2ModelConfigBuilder(AutoModelConfigBuilder):
+
+    @classmethod
+    def condition(cls, hf_config):
+        """config."""
+        return hf_config.model_type in ['deepseek_vl_v2']
+
+    @classmethod
+    def build(cls, hf_config, model_path: str = None, **kwargs):
+        """build deepseek-vl2."""
+
+        if hf_config.language_config.use_mla:
+            from .deepseek_v2 import DeepseekV2ModelConfigBuilder
+            cfg = DeepseekV2ModelConfigBuilder.build(hf_config.language_config, model_path, **kwargs)
+            cfg.hf_config = hf_config
+        else:
+            # deepseek-vl2-tiny use MHA, rather than MLA
```

Comment:
@CUHKSZzxy  `deepseek-ai/deepseek-vl2-tiny` failed with 
` ``
 File "/home/root/lmdeploy/lmdeploy/pytorch/models/deepseek_v2.py", line 121, in __init__
    config.kv_lora_rank + config.qk_rope_head_dim,
TypeError: unsupported operand type(s) for +: 'NoneType' and 'int'
```